org/mozilla/jss/ssl/SSLSocket.java

```diff
@@ -1596,7 +1596,7 @@ private static native boolean isFipsCipherSuiteNative(int ciphersuite)
      * Returns a list of cipher suites that are implemented by NSS.
      * Each element in the array will be one of the cipher suite constants
      * defined in this class (for example,
-     * <tt>TLS_RSA_WITH_AES_128_CBC_SHA</tt>).
+     * <code>TLS_RSA_WITH_AES_128_CBC_SHA</code>).
```

Comment:
I force pushed a change reverting the parenthesis to the original location.

Reply:
The changes you made are fine, ACK's feel free to push. Thanks.